registry/configuring_registry_storage/configuring-registry-storage-rhodf.adoc

```diff
@@ -0,0 +1,22 @@
+:_content-type: ASSEMBLY
+[id="configuring-registry-storage-rhodf"]
+= Configuring the registry for Red Hat OpenShift Data Foundation
+include::_attributes/common-attributes.adoc[]
+:context: configuring-registry-storage-rhodf
+
+toc::[]
+
+To configure the internal image registry on bare metal and vSphere to use {rh-storage-first} storage, you must install {rh-storage} and then configure your storage using Cepf or Noobaa.
```

Comment:
s/your storage using Cepf or Noobaa/image registry using Ceph or Noobaa

Reply:
Do we need link the ODF offical docs https://access.redhat.com/documentation/en-us/red_hat_openshift_data_foundation/4.10 ？

Reply:
OK，saw you have linked the https://access.redhat.com/documentation/en-us/red_hat_openshift_data_foundation/4.10/html-single/managing_and_allocating_storage_resources/index#configuring-image-registry-to-use-openshift-data-foundation_rhodf, thanks